WTWSMS/common/triggered_modifiers/MZ_triggered_modifiers.txt

```diff
@@ -124,3 +124,117 @@ commissioned_shahnameh_addition = {
 	zoroastrian_opinion = 10
 	monthly_character_prestige = 0.30
 }
+priscillian_carnivore = {
+	potential = {
+		religion = priscillianism
+	}
+	trigger = {
+		age = 16
+		religion = priscillianism
+		NOT = {
+			trait = vegetarian
+		}
+		AND = {
+			OR = {
+				is_theocracy = yes
+				controls_religion = yes
+				trait = monk
+				trait = nun
+				trait = cannibal_trait
+				has_character_modifer = cannibal
+				has_character_modifier = known_cannibal
+				has_job_title = job_spiritual
+			}
+			OR = {
+				trait = hunter
+				trait = falconer
+				NOT = {
+					trait = vegetarian
+				}
+			}		
+		}
+	}
+	icon = 60
+	monthly_character_piety = -1
+	same_religion_opinion = -20
+	monthly_character_prestige = -0.2
+}
+donamonaprisc_decadent = {
+	potential = {
+		OR = {
+			religion = monatist
+			religion = donatism
+			religion = priscillianism
+		}
+	}
+	trigger = {
+		age = 16
+		OR = {
+			religion = monatist
+			religion = donatism
+			religion = priscillianism
+		}
+		OR = {
+			trait = lustful
+			trait = decadent
+			trait = cruel
+			trait = greedy
+			trait = torturer
+			trait = drunkard
+			trait = impaler
+			trait = cannibal
+			trait = gluttonous
+			trait = corrupt
+			has_character_modifier = incestuous_adulterer
+			has_character_modifier = incestuous_adultereress
+			has_character_modifier = adulteress
+			has_character_modifier = adulterer
+			has_character_modifier = known_sodomite
+			has_character_modifier = adulterous_priest
+			has_character_modifier = cannibal
+			has_character_modifier = known_cannibal
+			has_character_modifier = known_murderer
+			has_character_focus = focus_seduction
+			has_character_focus = focus_carousing
+#			has_instances_of_character_modifier = {
+#				modifier = incestuous_adulterer
+#				factor = 1
+#			}
+#			has_instances_of_character_modifier = {
+#				modifier = incestuous_adulteress
+#				factor = 1
+#			}
+#			has_instances_of_character_modifier = {
+#				modifier = known_cannibal
+#				factor = 1
+#			}
+#			has_instances_of_character_modifier = {
+#				modifier = known_murderer
+#				factor = 1
+#			}
+#			AND = {
+#				has_instances_of_character_modifier = {
+#					modifier = adulterous_priest
+#					factor = 1
+#				}
+#				religion = donatism
+#			}
+		}
+	}
+	icon = 26
+	monthly_character_piety = -0.5
+	same_religion_opinion = -20
+	monthly_character_prestige = -0.2
+}
+priscillian_vegetarian = {
+	potential = {
+		religion = priscillianism
+	}
+	trigger = {
+		religion = priscillianism
+		trait = vegetarian
+	}
+	icon = 60
+	monthly_character_piety = 0.75
+	same_religion_opinion = 10
+}
```

Comment:
What do you need to figure out? Regardless it is not being kept as we can't afford destroying performance until one of these days.